src/devices/pioneer_ddj_400/mod.rs

```diff
@@ -20,11 +20,27 @@ pub const MIDI_DEVICE_DESCRIPTOR: &MidiDeviceDescriptor = &MidiDeviceDescriptor
 pub enum Deck {
     /// Left
     One,
+    /// Right
+    Two,
+}
 
+#[derive(Debug, Clone, Copy)]
+pub enum Channel {
```

Comment:
Why? The controller does not distinguish between *deck* and *channel*.

Reply:
:ok_hand: